new/analyse/spec/index.spec.ts

```diff
@@ -45,6 +53,24 @@ describe("analyse's Index", () => {
     expect(mockComposeAsync.mock.calls[0][1]).toBe(mockAddDependencies);
   });
 
+  test("3. extend package's dependencies ranges", () => {
+    // @ts-ignore
+    expect(mockComposeAsync.mock.calls[0][2]).toBe(mockExtendDependencyRanges);
+  });
+
+  test("4. creates package's dependencies tree", () => {
+    // @ts-ignore
+    expect(mockComposeAsync.mock.calls[0][3]).toBe(mockCreateDependencyTree);
+  });
+
+  test('calls "wait(1000)" at the end', () => {
```

Comment:
Well, to be honest. That one has really only been added to test the async setup. Why would we lose one second in every step? So no need to test it. And it can also be removed in one of the next PRs.